src/storage/simple.py

```diff
@@ -34,7 +34,7 @@ def save(self, epoch: int, modules: dict[str, nn.Module], metric: dict[str, floa
         '''
         if epoch % self.save_freq == 0:
             epoch_path = self.save_path / str(epoch)
-            epoch_path.mkdir(parents=True, exist_ok=True)
+            epoch_path.mkdir()
```

Comment:
Why do we remove parents=True, exist_ok=True?

Reply:
The parent directories are created in the `__init__`, but I think you might be right that I was too hasty to remove `exist_ok`. The logic behind the deletion was that there shouldn't be any folders there, but I forgot that we prompt the user for deletion if we detect that the folder already exists. Should we assume that if the user doesn't want to delete already existing folders then we can overwrite any files there, or should we exit and let the user backup data instead? If it's the latter then we can safely work without `exist_ok`, because there shouldn't be anything there in that case. Howerver, I'm not sure that the implementation complies with that logic, I will make sure it does.

Reply:
I suppose you're right that we don't need these params according to `init()` and program's logic.

However, I think we must always abort if the folder already exists. Program shouldn't be able to delete something itself even if it asks for it -- you can make a typo in the answer.

Reply:
Yeah nah, that's unlikely to make a typo between "y" and "n", so I think we're safe here, won't fix. And yes, our implementation does comply with what I proposed.